src/main/java/TasksManager.java

```diff
@@ -0,0 +1,133 @@
+import java.util.ArrayList;
+
+public class TasksManager {
+    private static final int EVENT_DEADLINE_TASK_NUMBER_OF_PARAMETERS = 2;
+
+    private ArrayList<Task> list;
+    private int numberOfTasks;
+
+    public TasksManager() {
+        this.list = new ArrayList<Task>();
+        this.numberOfTasks = 0;
+    }
+
+    public ArrayList<Task> getList() {
+        return this.list;
+    }
+
+    public boolean isEmpty() {
+        return list.isEmpty();
+    }
+
+    public int getNumberOfTasks() {
+        return this.numberOfTasks;
+    }
+
+    public void setNumberOfTasks(int numberOfTasks) {
+        this.numberOfTasks = numberOfTasks;
+    }
+
+    public void incrementNumberOfTasks() {
+        this.numberOfTasks += 1;
+    }
+
+    public boolean addTask(String text) {
+        // Create new Task object with text
+        boolean taskAddedSuccessfully;
```

Comment:
perhaps can rename this boolean variable to isTaskAdded because taskAddedSuccessfully sounds like a statement